app/controllers/comments_controller.rb

```diff
@@ -14,6 +19,10 @@ def create
   end
 
   def destroy
+    @notification = Notification.where(:comment_id == @comment.id)
```

Comment:
this isn't valid syntax
@notification = Notification.where(comment_id: @comment.id)

also why are you searching for it by comment id instead of notification id? Can't a comment have many notifications?